From e73c9a89825722b96042a8e6d9b298ed030780c2 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Wed, 18 May 2011 18:42:33 -0400
Subject: [PATCH 167/248] omap-merge-fix-plat-mach

omap merge fix plat mach

Temporary fixup for integration from linux omap to mainline tree.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
---
 arch/arm/mach-omap2/include/mach/trace-clock.h |    2 +-
 arch/arm/mach-omap2/trace-clock.c              |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-omap2/include/mach/trace-clock.h b/arch/arm/mach-omap2/include/mach/trace-clock.h
index 697bdda..cba12d0 100644
--- a/arch/arm/mach-omap2/include/mach/trace-clock.h
+++ b/arch/arm/mach-omap2/include/mach/trace-clock.h
@@ -9,7 +9,7 @@
 
 #include <linux/clk.h>
 #include <linux/timer.h>
-#include <plat/clock.h>
+#include <mach/clock.h>
 
 /*
  * Number of hardware clock bits. The higher order bits are expected to be 0.
diff --git a/arch/arm/mach-omap2/trace-clock.c b/arch/arm/mach-omap2/trace-clock.c
index 47e8fde..1f1cff9 100644
--- a/arch/arm/mach-omap2/trace-clock.c
+++ b/arch/arm/mach-omap2/trace-clock.c
@@ -14,7 +14,7 @@
 #include <linux/cpu.h>
 #include <linux/cpufreq.h>
 
-#include <plat/clock.h>
+#include <mach/clock.h>
 #include <asm/trace-clock.h>
 
 /* depends on CONFIG_OMAP_32K_TIMER */
-- 
1.7.0.4

